core/src/database/migrations.cpp

```diff
@@ -32,21 +32,41 @@
 
 namespace ledger {
     namespace core {
+        int getDatabaseMigrationVersion(soci::session& sql) {
+            int version = -1;
+
+            try {
+                soci::statement st = (sql.prepare << "SELECT version FROM __database_meta__ WHERE id == 0", soci::into(version));
```

Comment:
Could you remove the "==" please ? :trollface:

Reply:
Hehe, sure. Done.